### PR TITLE
Batch-Delete Invites (SST-12015)

### PR DESCRIPTION
**Pull request by clrobinson-traction**
### Description
Adding two new endpoints to support batch-deleting invites. Specifically, this is for the Appointments feature.

### Changelog
- New Model: `IdentifierList`
- New Endpoint: `POST /invites/batch_delete`
- New Endpoint: `GET /batches/:batch_id`

### CODE REVIEW

**Comment by tractionguest-integrations** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  Remember to use the `Squash and Merge` button to merge for single PRs!
    Always remember, your final commit message needs to start with a wrike ticket like, `[#1234] Your commit message`.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Nice! You've specified a versioning label :+1:</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  <a href="https://tractionguest.stoplight.io/docs/guest-api/branches/feature/batch-delete-invites/openapi.yml" target="_blank">View docs for this page</a> <em>(You'll need to have access to see this version of the docs)</em>
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7bded5d874d0a9697cd6c4495f0ca77459e104ef
</p>

**Review by baldys — DISMISSED** [review]

File: openapi.yml [hunk 1]
```diff
@@ -38,6 +38,32 @@ servers:
...
+      tags:
+        - Batches
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: ./models/BatchJob.v1.yaml
+        '404':
```
Comment (stevenallen05) [inline openapi.yml:59]: You'll probably want to change this to a wildcard:

```suggestion
        '4XX':
```

File: openapi.yml [hunk 2]
```diff
@@ -1786,6 +1812,33 @@ paths:
...
+        - Invites
+        - Batches
+      responses:
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: ./models/BatchJob.v1.yaml
+        '401':
```
Comment (stevenallen05) [inline openapi.yml:1828]: This should be a wildcard, too:

```suggestion
        '4XX':
```

**Review by stevenallen05 — CHANGES_REQUESTED** [review]

**Comment by stevenallen05** [comment]
Also, linting failed

**Comment by clrobinson-traction** [comment]
@stevenallen05 Linting should be fixed now.

**Review by baldys — APPROVED** [review]

**Comment by brandonjmckay** [comment]
@clrobinson-traction you have some conflicting files... 😭

**Pull request merged**